data_steward/analytics/cdr_ops/vocabulary_upload_qc.py

```diff
@@ -82,7 +97,12 @@
 # ''')
 # -
 
-# # The difference between the number of rows in each table of the dataset. All 'changes' should increase. https://github.com/OHDSI/Vocabulary-v5.0/releases
+#
+# # Tables row count comparison
+#
+# > - The difference between the number of rows in each table of the datasets.  <br>
+# > - All 'changes' should increase. Expected changes are found in the [release notes](https://github.com/OHDSI/Vocabulary-v5.0/releases).  <br>
```

Comment:
They are almost always increasing.  We have had at least one instance where a vocabulary got merged into another vocabulary and the vocabulary count dropped from 83 to 82.  Again, this is an edge case and as a general rule we expect row counts to increase as you mention.